tests/e2e/utils/pw-setup.js

```diff
@@ -189,6 +190,76 @@ export const installPluginFromRepository = ( page ) =>
 		} )();
 	} );
 
+/**
+ * Helper function to update the plugin.
```

Comment:
Maybe rephrase this to something like:

> Helper function to download and install the WooCommerce Subscriptions plugin from its latest GitHub release.

Reply:
I took the time to fix all docstrings on this file. https://github.com/woocommerce/woocommerce-gateway-stripe/pull/2522/commits/7f9d4ed5e7fba1fafb1425bf1feb77dc8864854b